packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { startCase } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, CheckboxControl, Modal } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState, useCallback } from '@wordpress/element';
+
+const EntitiesSavedStatesCheckbox = ( {
+	id,
+	name,
+	changes: { rawRecord, edits },
+	checked,
+	setCheckedById,
+} ) => (
+	<CheckboxControl
+		label={ `${ startCase( name ) }: "${ rawRecord.title || rawRecord.name }"` }
+		help={ `Changed Properties: ${ Object.keys( edits ).join( ', ' ) }.` }
```

Comment:
I think that part is not really useful and just confusing. I'd suggest just removing it. (the changed properties helper)

Reply:
Agreed, at least until we have a better UI for it.